src/components/ShowList.jsx

```diff
@@ -50,26 +50,36 @@ function ShowList() {
               aria-label="clear filter text"
               className="clear-button"
             >
-              <i class="fas fa-times"></i>
+              <i className="fas fa-times"></i>
             </button>
           )}
+          {actionMessage ? (
+            <p className="success-message">{actionMessage}</p>
+          ) : null}
```

Comment:
If we are deleting the very last item, this message won't show up since it depends on there being items in the list. We should be able to display the message regardless if there are items or not.